src/XMakeCommandLine/Microsoft.Build.CommonTypes.xsd

```diff
@@ -168,6 +168,51 @@ elementFormDefault="qualified">
       </xs:complexContent>
     </xs:complexType>
   </xs:element>
+  <xs:element name="PackageReference" substitutionGroup="msb:Item">
+    <xs:annotation>
+      <xs:documentation>
+        <!-- _locID_text="PackageReference" _locComment="" -->Reference to a package
+      </xs:documentation>
+    </xs:annotation>
+    <xs:complexType>
+      <xs:complexContent>
+        <xs:extension base="msb:SimpleItemType">
+          <xs:sequence minOccurs="0" maxOccurs="unbounded">
+            <xs:choice>
+              <xs:element name="Version">
```

Comment:
If I typed `<PackageReference Include="" ` will intellisense show Version as a possible attribute? We should make it so that it doesn't guide people towards using Version as an element instead of an attribute.